packages/dds/tree/src/tree/anchorSet.ts

```diff
@@ -264,16 +365,22 @@ class PathNode implements UpPath {
         assert(childIndex !== undefined, "child must be parented to be removed");
         field?.splice(childIndex, 1);
         if (field?.length === 0) {
+            this.afterEmptyField(key);
+        }
+    }
+
+    public afterEmptyField(key: FieldKey): void {
```

Comment:
Some docs on this method would be welcome. What does it do? Are users of the class obligated to call it? Maybe renaming it to include a verb would be helpful too.